docs/cep/metadata.md

```diff
@@ -0,0 +1,83 @@
+---
+sidebar_position: 10
+title: Stream Worker Metadata
+---
+
+Stream workers have both required and optional metadata fields. This page explains what they are and how to use them.
+
+## Example Metadata
+
+This is an example of what metadata fields might look like. Each field is indicated with the `@App:` prefix. Individual metadata fields are described below.
+
+```js
+@App:name("New1")
+@App:description("My description")
+@App:qlVersion("2")
+```
+
+## Name (Required)
+
+Every stream worker requires a unique name to be valid. Names cannot include spaces or special symbols such as `*`. After you save the stream worker, you cannot edit the name.
+
+The name format is:
+
+```js
+@App:name("MyStreamWorker")
+```
+
+## Description
+
+The description is an optional string of text that tells other users (or your future self) about the stream worker. It should not be more than one to three sentences.
+
+The description format is:
+
+```js
+@App:description("This stream worker prints Hello World every hour, on the hour.")
+```
+
+## Query Language Version (Required)
+
+`qlVersion` is used for backwards compatibility with deprecated stream workers. All new stream workers use version 2 as shown below.
+
+```js
+@App:qlVersion("2")
+```
+
+## Docstrings and Other Information
```

Comment:
Can be mentioned single line comment. It starts from `--`. Here an example:
```
-- 'eventTimestamp()' returns the timestamp of the processed/passed event.
```

Reply:
Yes, that's a good point. Thanks, @dlazarenko-macrometa .
@diana-macrometa - Since Stream Query language is SQL-like, it supports SQL line and block comments:

- The line comment is `--` (double dashes), e.g. 
```
-- This is my comment
```
- Block comment is `/* .... */` but not `/** ... **/`, e.g.
```
/* 
This
is
my comment.
*/
```

Reply:
Are the single-line comments metadata?

Reply:
Updated